modules/creating-multiple-ingress-through-single-alb.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/multiple-ingress-through-single-alb.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-multiple-ingress-through-single-alb_{context}"]
+= Creating multiple ingresses through single ALB
+
+The user can route the traffic to multiple Ingresses through a single ALB.
+
+.Procedure
+
+. Create an `IngressClassParams` resource:
++
+[source,yaml]
+----
+apiVersion: elbv2.k8s.aws/v1beta1 <1>
+kind: IngressClassParams
+metadata:
+  name: single-lb-params <2>
+spec:
+  group:
+    name: single-lb <3>
+----
+
+<1> Defines the API group and version of the `IngressClassParams` resource.
+<2> Specifies the name of the `IngressClassParams` resource.
+<3> Specifies the name of the `IngressGroup`. All Ingresses of this class belong to this `IngressGroup`.
+
+. Create an `IngressClass`.
++
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1 <1>
+kind: IngressClass
+metadata:
+  name: single-lb <2>
+spec:
+  controller: ingress.k8s.aws/alb <3>
+  parameters:
+    apiGroup: elbv2.k8s.aws <4>
+    kind: IngressClassParams <5>
+    name: single-lb <6>
+----
+
+<1> Defines the API group and the version of the `IngressClass` resource.
+<2> Specifies the name of the `IngressClass`.
+<3> Defines the controller name, common for all `IngressClasses`. The `aws-load-balancer-controller` reconciles the controller.
+<4> Defines the API group of the `IngressClassParams` resource.
+<5> Defines the resource type of the `IngressClassParams` resource.
+<6> Defines the name of the `IngressClassParams` resource.
+
+. Create the `Ingress` resources:
++
+[source,yaml]
+----
+kind: Ingress
```

Comment:
Per the OpenShift Docs guideline for [API object required fields](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#required-fields), include `apiVersion`